docs/pages/includes/config-reference/proxy-service.yaml

```
@@ -161,3 +161,18 @@ proxy_service:
         # An absolute path to the file containing the API token for OpenAI.
         # This is required for Assist functionality.
         api_token_path: /etc/teleport/openai_key
+
+    # Configuration for the built-in version server for agent automatic updates.
+    # If you leave this configuration unset, a default channel is created and
+    # serves the teleport version run by the proxy.
```

```suggestion
    # serves the Teleport version run by the Proxy Service.
```
Clarifying that this is the Teleport Proxy Service, not another proxy the user might have running in their infrastructure.